test/Microsoft.ML.Benchmarks/StochasticDualCoordinateAscentClassifierBench.cs

```diff
@@ -123,39 +132,34 @@ public void TrainSentiment()
             _consumer.Consume(predicted);
         }
 
-        [GlobalSetup(Targets = new string[] { nameof(PredictIris), nameof(PredictIrisBatchOf1), nameof(PredictIrisBatchOf2), nameof(PredictIrisBatchOf5) })]
+        [GlobalSetup(Target = nameof(PredictIris))]
         public void SetupPredictBenchmarks()
         {
+            var env = new MLContext(seed: 1, conc: 1);
             _trainedModel = Train(_dataPath);
-            _consumer.Consume(_trainedModel.Predict(_example));
+            _predictionEngine = _trainedModel.CreatePredictionEngine<IrisData, IrisPrediction>(env);
+            _consumer.Consume(_predictionEngine.Predict(_example));
 
-            var testData = new Legacy.Data.TextLoader(_dataPath).CreateFrom<IrisData>(useHeader: true);
-            var evaluator = new ClassificationEvaluator();
-            _metrics = evaluator.Evaluate(_trainedModel, testData);
+            var reader = new TextLoader(env,
+                    columns: new[]
+                    {
+                            new TextLoader.Column("Label", DataKind.R4, 0),
+                            new TextLoader.Column("SepalLength", DataKind.R4, 1),
+                            new TextLoader.Column("SepalWidth", DataKind.R4, 2),
+                            new TextLoader.Column("PetalLength", DataKind.R4, 3),
+                            new TextLoader.Column("PetalWidth", DataKind.R4, 4),
+                    },
+                    hasHeader: true
+                );
 
-            _batches = new IrisData[_batchSizes.Length][];
-            for (int i = 0; i < _batches.Length; i++)
-            {
-                var batch = new IrisData[_batchSizes[i]];
-                _batches[i] = batch;
-                for (int bi = 0; bi < batch.Length; bi++)
-                {
-                    batch[bi] = _example;
-                }
-            }
+            IDataView testData = reader.Read(_dataPath);
+            IDataView scoredTestData = _trainedModel.Transform(testData);
+            var evaluator = new MultiClassClassifierEvaluator(env, new MultiClassClassifierEvaluator.Arguments());
+            _metrics = evaluator.Evaluate(scoredTestData, DefaultColumnNames.Label, DefaultColumnNames.Score, DefaultColumnNames.PredictedLabel);
         }
 
         [Benchmark]
-        public float[] PredictIris() => _trainedModel.Predict(_example).PredictedLabels;
-
-        [Benchmark]
-        public void PredictIrisBatchOf1() => Consume(_trainedModel.Predict(_batches[0]));
-
-        [Benchmark]
-        public void PredictIrisBatchOf2() => Consume(_trainedModel.Predict(_batches[1]));
-
-        [Benchmark]
-        public void PredictIrisBatchOf5() => Consume(_trainedModel.Predict(_batches[2]));
```

Comment:
`BatchPredictionEngine` is not part of new API, so removing any code related to batch predictions.